Sources/XcodeProj/Project/XcodeProj.swift

```diff
@@ -8,6 +8,9 @@ public final class XcodeProj: Equatable {
     /// Project workspace
     public var workspace: XCWorkspace
 
+    /// The  path to the `.xcodeproj` directory.
+    private let projectPath: Path?
```

Comment:
```suggestion
    private let path: Path?
```

I'd align with the value of the `init` parameter